source/patterns/@aws-solutions-constructs/core/test/elasticsearch-helper.test.ts

```diff
@@ -170,8 +170,6 @@ test('Test VPC with 1 AZ, Zone Awareness Disabled', () => {
 });
 
 test('Test VPC with 2 AZ, Zone Awareness Enabled', () => {
-  // Providing a fake account number and region enables us to
-  // get expected behavior of VPC default properties
   const stack = new Stack(undefined, undefined, {
```

Comment:
If we want 2 AZs, why are we providing an environment here?

Reply:
Thought adding props will add better clarity but since we have the comment then this isn't necessary. Will remove the prop in new code